src/Framework/AspNetCoreAnalyzers/src/CodeFixes/DetectMismatchedParameterOptionalityFixer.cs

```diff
@@ -13,6 +14,7 @@
 
 namespace Microsoft.AspNetCore.Analyzers.RouteHandlers.Fixers;
 
+[ExportCodeFixProvider(LanguageNames.CSharp), Shared]
```

Comment:
What does the `Shared` indicate here?

Reply:
@captainsafia This will lead to a single instance of the codefix provider being returned when requested. This is safe because codefix providers are stateless. The benefit is reducing memory.